bundle/README.md

```diff
@@ -1,21 +1,20 @@
 # Livepatch server on-premises bundle
 
-This juju bundle deploys a [livepatch](https://ubuntu.com/security/livepatch)
-server for serving patches to machines running livepatch client.
+[![CharmHub Badge](https://charmhub.io/canonical-livepatch-onprem/badge.svg)](https://charmhub.io/canonical-livepatch-onprem)
+[![Release](https://github.com/canonical/livepatch-machine-charm/actions/workflows/publish_bundle.yaml/badge.svg)](https://github.com/canonical/livepatch-machine-charm/actions/workflows/publish_bundle.yaml)
```

Comment:
This release action doesn't actually work as expected, that's something to be fixed eventually. Not sure how you want to handle the badge with that in mind.

Reply:
As discussed, I've now dropped the badge from this PR.